### DISCUSSION TITLE
How to have the full example website as my own initial website?

### DISCUSSION THREAD

**Discussion by bghjmn32** [lxndrblz/anatole#194] [Q&A]
Hello, 

I am new to this area and lack experience, I am trying to create my blog based on your theme, precisely speaking, the whole "example website".

What I have done is:

> move all folder in "example Site" to my own Blog folder
> replace original config.toml with the "example Site" one.
> reaplce the original "baseURL" with my own github repositoriy Url which is https://bghjmn32.github.io/hongyiluo.github.io/
> use `hugo -d docs` to generate docs and use the "docs" folder branch as the source.

After these, I checked the website on my local explore, everything is fine.

![Pic1](https://user-images.githubusercontent.com/23047364/116924601-649c5a80-ac58-11eb-94b1-801eef28365a.png)

But you may also find it out from the link I posted in step 3. 

![pic2](https://user-images.githubusercontent.com/23047364/116924893-be048980-ac58-11eb-8d56-fd687f9e5690.png)

It seems I lost all css and js results. 

Do you have any hints or advice for me to fix this ?

Thank you very much!

**Comment by lxndrblz** [comment]
@bghjmn32 I've once more checked your deployment 3696f25 and the page appears correctly now. What did you change?

![image](https://user-images.githubusercontent.com/12200062/116926973-f1441a00-ac52-11eb-972e-a3ffc08b5626.png)

**Reply by bghjmn32** [reply]
Hello, 

Thanks for your quick answer.

I actually find out there are 2 "config.toml", one is in the main folder and the other is in the "\config\_default" (because I copied all files from the "exampleSite" ), I only modified the first one before. So I am wondering if it's the issue.

And seems it is. Honestly, I actually don't know if they both work or only one of them work....

**Reply by lxndrblz** [reply]
Hi @bghjmn32,

You can either setup the config structure using the `config_default` folder and placing the individual sections in there. Otherwise, you can also create a single `config.toml` where you mark the the individual sections. Both approaches are valid, but you should stick to one of them. In your case I would use the `config.toml` in the main folder, which gets created automatically if you create a new hugo site.

**Reply by bghjmn32** [reply]
Hello, Thank you for your answer.

I also want to ask you if there is any way to change the width of the 2 col ? Especially for the left one ( with the profile pic), I would like it to be thinner to left more reading space for the right article part.

**Reply by lxndrblz** [reply]
@bghjmn32 Yes that is in progress: Please follow issue #180  for updates. 

For now, you can also change it manually by `.content`, `.page-top` and `.sidebar` with your values of choice.
